frontend/ui/widget/keyvaluepage.lua

```diff
@@ -597,7 +597,15 @@ function KeyValuePage:_populateItems()
             end
         end
     end
-    self.page_info_text:setText(T(_("Page %1 of %2"), self.show_page, self.pages))
+
+    -- update page information
+    if self.pages > 1 then
+        self.page_info_text:setText(T(_("Page %1 of %2"), self.show_page, self.pages))
+    else
+        self.page_info_text:setText("Single page");
```

Comment:
I know page 1 of 1 looks a bit silly but this

a. isn't much less silly
b. is inconsistent

Reply:
I agree:
"Single page" is more prose and will attract your eyes more easily than a consistent "Page 1 of 1"